great-idea-website/css/index.css

```diff
@@ -63,4 +63,122 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+hr {
+	width: 80%;
+}
+.main-content {
+	margin: 0 20%;
+}
+
+nav {
+	width: 70%;
+	display: inline-block;
+	margin: 0 0 0 5%;
+
+}
+nav a {
+	text-decoration: none;
+	display: inline-block;
+	margin: 5%;
```

Comment:
The design file has the links' font color as black. You may add this property here to suppress the default color for (_unclicked_) links, which is blue.